ui/core/components/individual_sim_ui/apl_values.ts

```diff
@@ -1129,38 +1128,62 @@ const valueKindFactories: { [f in ValidAPLValueKind]: ValueKindConfig<APLValueIm
-		fields: [AplHelpers.unitFieldConfig('sourceUnit', 'aura_sources'), AplHelpers.actionIdFieldConfig('auraId', 'auras', 'sourceUnit')],
+		newValue: () => APLValueAuraIsActive.create({ includeReactionTime: true }),
+		fields: [
+			AplHelpers.unitFieldConfig('sourceUnit', 'aura_sources'),
+			AplHelpers.actionIdFieldConfig('auraId', 'auras', 'sourceUnit'),
+			AplHelpers.booleanFieldConfig('includeReactionTime', 'Include Reaction Time', {
+				labelTooltip: 'If checked, will use the configured reaction time.',
+			}),
+		],
 	}),
 	auraIsActiveWithReactionTime: inputBuilder({
-		label: 'Aura Active (with Reaction Time)',
+		label: '[DEPRECATED] Aura Active (With Reaction Time)',
 		submenu: ['Aura'],
-		shortDescription:
-			'<b>True</b> if the aura is currently active AND it has been active for at least as long as the player reaction time (configured in Settings), otherwise <b>False</b>.',
-		newValue: APLValueAuraIsActiveWithReactionTime.create,
+		shortDescription: 'All Aura checks now contain Reaction Time logic, so this is no longer needed. Use <b Aura Active</b> instead.',
+		newValue: () => APLValueAuraIsActive.create({ includeReactionTime: true }),
 		fields: [AplHelpers.unitFieldConfig('sourceUnit', 'aura_sources'), AplHelpers.actionIdFieldConfig('auraId', 'auras', 'sourceUnit')],
 	}),
-	auraIsInactiveWithReactionTime: inputBuilder({
-		label: 'Aura Inactive (with Reaction Time)',
+	auraIsInactive: inputBuilder({
+		label: 'Aura Inactive',
 		submenu: ['Aura'],
 		shortDescription:
 			'<b>True</b> if the aura is not currently active AND it has been inactive for at least as long as the player reaction time (configured in Settings), otherwise <b>False</b>.',
-		newValue: APLValueAuraIsInactiveWithReactionTime.create,
+		newValue: () => APLValueAuraIsInactive.create({ includeReactionTime: true }),
+		fields: [
+			AplHelpers.unitFieldConfig('sourceUnit', 'aura_sources'),
+			AplHelpers.actionIdFieldConfig('auraId', 'auras', 'sourceUnit'),
+			AplHelpers.booleanFieldConfig('includeReactionTime', 'Include Reaction Time', {
+				labelTooltip: 'If checked, will use the configured reaction time.',
+			}),
+		],
+	}),
+	auraIsInactiveWithReactionTime: inputBuilder({
+		label: '[DEPRECATED] Aura Inactive (With Reaction Time)',
+		submenu: ['Aura'],
+		shortDescription: 'All Aura checks now contain Reaction Time logic, so this is no longer needed. Use <b Aura Inactive</b> instead.',
+		newValue: () => APLValueAuraIsInactive.create({ includeReactionTime: true }),
 		fields: [AplHelpers.unitFieldConfig('sourceUnit', 'aura_sources'), AplHelpers.actionIdFieldConfig('auraId', 'auras', 'sourceUnit')],
 	}),
 	auraRemainingTime: inputBuilder({
 		label: 'Aura Remaining Time',
 		submenu: ['Aura'],
 		shortDescription: 'Time remaining before this aura will expire, or 0 if the aura is not currently active.',
-		newValue: APLValueAuraRemainingTime.create,
+		newValue: () => APLValueAuraRemainingTime.create({ includeReactionTime: true }),
```

Comment:
The `includeReactionTime` input is not actually used in the back-end for this one, so it should either be removed from the proto or implemented in the back-end.